CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 This file includes a list of notable changes to this project.
 
+## [0.5.0](https://github.com/neuralet/smart-social-distancing/releases/tag/0.6.0)
+Released on 2021-03-16.
+
+#### Added:
+
+* Draw Region of Interest contour in live feed ([#136](https://github.com/neuralet/smart-social-distancing/pull/136))
+* List overview of camera configuration ([#138](https://github.com/neuralet/smart-social-distancing/pull/138))
+* In/Out Boundary endpoint configuration ([#135](https://github.com/neuralet/smart-social-distancing/pull/135))
```

Comment:
Wouldn't it make more sense to have just one entry for In/Out with links to the 3 PRs?